test/default.yaml

```diff
@@ -0,0 +1,2 @@
+default-docker:
+  use-sigstore-attachments: true
```

Comment:
will this involve a change in packaging? do we need this file to be present now? If possible, I'd prefer we didn't need this and optionally opted into it

Reply:
The MCO will do that for us.